.github/workflows/deploy.yml

```diff
@@ -16,6 +16,10 @@ jobs:
       - uses: actions/checkout@v3
         with:
           token: ${{ secrets.GAMITEE_BOT_COMMIT_ACCESS_TOKEN }}
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: package.json
```

Comment:
🤩

Reply:
I see logic in calling all setup Actions first, but seems more security-oriented to cal `actions/setup-node` below, just before we actually use `npm` (i.e. `node`).